minitwit_admin/src/components/TwitListComponent.vue

```diff
@@ -94,21 +100,81 @@ export default {
   components: {},
   emits: ["onClick"],
   setup(props, context) {
-    const handleItemClick = (item) => {
-      context.emit("onClick", item);
-    };
+    const store = inject("store");
+    const { getFollowers } = useFollowers();
+    const { getLoggedInUser } = useUsers();
+    const followers = getFollowers();
+    const loggedInUser = getLoggedInUser();
+
+    const isAlreadyFollowed = (authorId) => {
+      return followers.value.some(entry => entry.whomId === authorId)
+    }
+
+    const handleItemClick = (item) => context.emit("onClick", item);
+    const followUser = (userId) => store.followers.actions.followUser(userId);
+
     return {
       handleItemClick,
+      loggedInUser,
+      followUser,
+      isAlreadyFollowed
     };
   },
 };
 </script>
 
 <style lang="scss" scoped>
+@import "../genericStyles.scss"; 
+@import "../_variables.scss"; 
+
 #TwitListComponent {
-  .icon {
-    &:hover {
-      cursor: pointer;
+  margin: 2rem auto;
+
+  .va-list {
+    padding: 0 0 2rem 0 !important;
+    
+    .va-list-item {
+      background-color: $twit-background !important;
+
+      @include roundedBorders;
+      @include shadow;
+      box-shadow: unset;
+      > * {
+        display: block !important;
+
+        .va-list-item-section {
+          &.content {
+            flex-basis: 90%;
+            text-align: left;
+          }
+
+          &.actionButtonsWrapper {
+            @include roundedBorders;
+            @include shadow;
+            display: flex;
+            flex-direction: row;
+            margin: auto 0.5rem;
+            background-color: $twit-btn-background;
+            box-shadow: rgba(0, 0, 0, 0.1) 0px 10px 15px -3px, rgba(0, 0, 0, 0.05) 0px 4px 6px -2px;
```

Comment:
@include shadow already includes box-shadow, just checking if this is intentional here